code/datums/actions/order_action.dm

```diff
@@ -156,12 +163,15 @@
 	action_icon_state = "rally"
 	verb_name = "rally to"
 	arrow_type = /obj/screen/arrow/rally_order_arrow
+	visual_type = /obj/effect/temp_visual/order/rally_order
+
+/datum/action/innate/order/rally_order/personal
 	keybind_signal = COMSIG_KB_RALLYORDER
```

Comment:
Can you explain this?

Reply:
personal meaning humans with the skill level gets it and makes them say a message but overwatch computers dont need that